src/core/BUILD

```diff
@@ -7,7 +7,7 @@ genrule(
         "version": ["//:version"],
     },
     outs = ["versioned.go"],
-    cmd = "sed \"s/1.0.9999/`cat $SRCS_VERSION`/\" $SRCS_GO > \"$OUT\"",
+    cmd = "sed \"s/1.0.9999/`cat $SRCS_VERSION`/g\" $SRCS_GO > \"$OUT\"",
```

Comment:
Can't we use go definitions to just set this value?

Reply:
Ah yeah good idea. Have done.

Also thought about using embedding which would mean it was accurate even for `go get` but this is a little easier...